compose-combined.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  formulosity:
+    restart: unless-stopped
+    image: formulosity:latest
+    build:
+      context: .
+      additional_contexts:
+        api: ./api
+        ui: ./ui
+      args:
+        - NEXT_PUBLIC_CONSOLE_API_ADDR=http://127.0.0.1:8080
+    ports:
+      - "3000:3000"
+      - "8080:8080"
+    init: true
+    environment:
+      - LOG_LEVEL=debug
+      - DATABASE_TYPE=sqlite # postgres|sqlite
+      - DATABASE_URL=/data/db/formulosity.db
+      - SURVEYS_DIR=/data/surveys
+      - CONSOLE_API_ADDR=http://127.0.0.1:8080
```

Comment:
We should also add NEXT_PUBLIC_CONSOLE_API_ADDR I think.

Reply:
in your `compose.yml`  `NEXT_PUBLIC_CONSOLE_API_ADDR`  also only appears as a build arg 
do we need it as an actual env variable in the container?